c++/CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 project("Cap'n Proto" CXX)
 cmake_minimum_required(VERSION 3.1)
-set(VERSION 0.7-dev)
+set(RELEASE_VERSION 0.7)
```

Comment:
But yeah, I can revert this change. Give me a moment and I'll push a replacement patch.